client/src/ui/components/hints/WorldStructures.tsx

```
@@ -118,33 +128,31 @@ const HyperstructureCompletionTable = () => {
-                    size="lg"
-                  />
-                  <ResourceCost
-                    className="truncate mb-1"
-                    resourceId={cost.resource}
-                    amount={cost.max_amount}
-                    size="lg"
-                  />
+          <div className="flex flex-col">
+            {completionCosts.map(({ resource, min_amount, max_amount }) => {
+              return (
+                <div className="flex px-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)